cache/lru_cache.cc

```diff
@@ -111,8 +112,9 @@ void LRUHandleTable::Resize() {
 
 LRUCacheShard::LRUCacheShard(
     size_t capacity, bool strict_capacity_limit, double high_pri_pool_ratio,
-    double low_pri_pool_ratio, bool use_adaptive_mutex,
-    CacheMetadataChargePolicy metadata_charge_policy, int max_upper_hash_bits,
+    double low_pri_pool_ratio, bool use_compressed_secondary_cache,
```

Comment:
Can you explain why `use_compressed_secondary_cache` is needed?

Reply:
I will add SecondaryCache::SupportForceErase() and remove this.